src/prefect/task_server.py

```diff
@@ -124,6 +138,9 @@ async def stop(self):
         raise StopTaskServer
 
     async def _subscribe_to_task_scheduling(self):
+        logger.debug(
+            f"Subscribing to tasks: {' | '.join(t.task_key.split('.')[-1] for t in self.tasks)}"
+        )
```

Comment:
You know, this should probably be `INFO` level because it could be very useful for people to see the exact task keys that will be served